cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -1,4 +1,13 @@
 {-# LANGUAGE DataKinds #-}
```

Comment:
I'm not sure how I feel about pulling all of the `LocalStateQuery` implementations into this CLI module. These still seem like API things, even if it is the old API.

Reply:
I'm porting the `Cardano.Api.LocalChainSync` to the new api in-place. We'll see if we can port the other queries to use the new api, whether they live in the cli or otherwise.

Reply:
I disagree, I think most of `LocalStateQuery` belongs in the cli and we should strip out the relevant bits that belong in `cardano-api`